swift_browser_ui/server.py

```diff
@@ -238,40 +237,37 @@ def run_server_secure(
     logger.debug("Setting up SSL context for the server.")
     sslcontext = ssl.create_default_context(purpose=ssl.Purpose.CLIENT_AUTH)
     cipher_str = (
-        "ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE" +
-        "-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA" +
-        "-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:DHE-RSA-AES128-GCM" +
-        "-SHA256:DHE-RSA-AES256-GCM-SHA384"
-    )
-    logger.debug(
-        "Setting following ciphers for SSL context: \n%s",
-        cipher_str
+        "ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE"
+        + "-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA"
+        + "-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:DHE-RSA-AES128-GCM"
+        + "-SHA256:DHE-RSA-AES256-GCM-SHA384"
     )
+    logger.debug("Setting following ciphers for SSL context: \n%s", cipher_str)
     sslcontext.set_ciphers(cipher_str)
     sslcontext.options |= ssl.OP_NO_TLSv1
     sslcontext.options |= ssl.OP_NO_TLSv1_1
     logger.debug("Loading cert chain.")
     sslcontext.load_cert_chain(cert_file, cert_key)
     aiohttp.web.run_app(
         app,
-        access_log=aiohttp.web.logging.getLogger('aiohttp.access'),
-        port=setd['port'],  # type: ignore
+        access_log=aiohttp.web.logging.getLogger("aiohttp.access"),
+        port=setd["port"],  # type: ignore
         ssl_context=sslcontext,
     )
 
 
 def run_server_insecure(
-        app: typing.Coroutine[typing.Any, typing.Any, aiohttp.web.Application]
+    app: typing.Coroutine[typing.Any, typing.Any, aiohttp.web.Application]
 ) -> None:
     """Run the server without https enabled."""
     aiohttp.web.run_app(
         app,
-        access_log=aiohttp.web.logging.getLogger('aiohttp.access'),
-        port=(setd['port'])  # type: ignore
+        access_log=aiohttp.web.logging.getLogger("aiohttp.access"),
+        port=(setd["port"]),  # type: ignore
```

Comment:
we could make this `int` instead of ignoring type

Reply:
The port is an integer in the dictionary – the settings dictionary could be moved over to a TypedDict to remove this issue, which would be a better solution in my opinion. This, however, would require dropping support for Python < 3.8 (TypedDict comes built-in from 3.8 onwards).
Forcing the integer with `int()` only generates a different mypy error.

Reply:
i am fine with dropping support for <3.8, but would postpone the Typedict for a different issue